scripts/tail-logs

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
Can this script get a note in the readme?

Reply:
done, thanks